libchisel/src/binaryenopt.rs

```diff
@@ -0,0 +1,107 @@
+/*
+ * Binaryen optimiser.
+ */
+
+use std::collections::HashMap;
+
+use super::{ModuleError, ModulePreset, ModuleTranslator};
+use parity_wasm::elements::*;
```

Comment:
Not to be pedantic but going forward I'd like to order the `use` statements with `std` first, followed by external imports, followed by local modules.

Reply:
Suggest to create a PR which rectifies that for the entire repo.